databases/week2/dbWeek2Part3.sql

```diff
@@ -0,0 +1,22 @@
+USE hyfHomeworkWeek1;
+SELECT * FROM task;
+SELECT * FROM status;
+SELECT * FROM user;
+
+
+--Part 3: More queries
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.*
+FROM task
+INNER JOIN user
+ON user.id
+
```

Comment:
Your query isn't complete and has a missing condition. I'm not quite sure about columns names but here is an example of query I'd use to get a result:

SELECT task.*
FROM task
INNER JOIN user
ON task.user_id = user.id
WHERE user.email LIKE '%@spotify.com';